File: C mas/InformacionUsuario.cpp
#include <iostream> // Para entrada y salida de datos
#include <string>   // Para manejar cadenas de texto
using namespace std;

int main()
{
    string nombre, apellido, profesion; // Variables para almacenar los datos del usuario
    int anioNacimiento, edad;

    // Solicitar al usuario su nombre completo
    cout << "Escriba sus nombres completos: ";
    getline(cin, nombre); // Leer toda la línea con espacios incluidos

    // Solicitar su apellido completo
    cout << "Escriba sus apellidos completos: ";
    getline(cin, apellido);

    // Solicitar su profesión
    cout << "Escriba su profesión: ";
    getline(cin, profesion);

    // Solicitar su año de nacimiento
    cout << "Escriba su año de nacimiento: ";
    cin >> anioNacimiento;

    // Calcular la edad en el año 2025
    edad = 2025 - anioNacimiento;

    // Mostrar el resultado
    cout << "El(La) " << profesion << " " << nombre << " " << apellido << " tiene " << edad << " años." << endl;

    return 0;
}

File: Java/InformacionUsuario.java
import java.util.Scanner;  // Importamos Scanner para leer los datos del usuario

public class InformacionUsuario {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);  // Creamos un objeto Scanner para entrada de datos
        
        // Solicitar al usuario su nombre completo
        System.out.print("Escriba sus nombres completos: ");
        String nombre = scanner.nextLine();
        
        // Solicitar su apellido completo
        System.out.print("Escriba sus apellidos completos: ");
        String apellido = scanner.nextLine();
        
        // Solicitar su profesión
        System.out.print("Escriba su profesión: ");
        String profesion = scanner.nextLine();
        
        // Solicitar su año de nacimiento
        System.out.print("Escriba su año de nacimiento: ");
        int anioNacimiento = scanner.nextInt();
        
        // Calcular la edad en el año 2025
        int edad = 2025 - anioNacimiento;

        // Mostrar el resultado
        System.out.println("El(La) " + profesion + " " + nombre + " " + apellido + " tiene " + edad + " años.");

        scanner.close(); // Cerramos el Scanner
    }
}
